MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -6990,14 +6963,12 @@ private void lastwd()
         {
             var stringPointerBase = GetParameterPointer(0);
 
-            var stringToParse = Encoding.ASCII.GetString(Module.Memory.GetString(stringPointerBase));
+            var stringToParse = Encoding.ASCII.GetString(Module.Memory.GetString(stringPointerBase, stripNull: true));
 
-            if (!Module.Memory.TryGetVariablePointer("lastwd-buffer", out var lastwdBufferPointer))
-                lastwdBufferPointer = Module.Memory.AllocateVariable("lastwd-buffer", 0xFF);
+            var lastWordIndex = stringToParse.LastIndexOf(' ');
+            lastWordIndex++; //Start on next character after space
```

Comment:
LastIndexOf could return -1 if not found, but I guess that's OK since the following increment will just move it back to 0?

Reply:
Correct -- That was the idea, where if it's not found it just returns a pointer to the beginning of the string (same pointer passed in)